web/client/components/widgets/widget/InfoPopover.jsx

```diff
@@ -51,13 +56,27 @@ class InfoPopover extends React.Component {
             </Popover>
         );
     }
-
+    renderContent() {
+        return (<Glyphicon
+            ref={button => {
+                this.target = button;
+            }}
+            className={`text-${this.props.bsStyle}`}
+            glyph={this.props.glyph} />);
+    }
     render() {
         return (
             <span className="mapstore-info-popover">
-                <OverlayTrigger trigger={['hover', 'focus']} placement={this.props.placement} overlay={this.renderPopover()}>
-                    <Glyphicon className={`text-${this.props.bsStyle}`} glyph="question-sign"/>
-                </OverlayTrigger>
+                {this.props.trigger
```

Comment:
empty array are considered true value. is it what intended here?
maybe this is better this.props.trigger && this.props.trigger.length

Reply:
I wanted thits semantics:
 -  trigger={false} to show always the popup
 - As an alternative trigger=['click', 'focus', 'hover'] to specify when to trigger. Maybe I should document it